dev_utils/docker-compose.yml

```diff
@@ -90,18 +108,64 @@ services:
       - SERVER_JWTPUBEYURL=https://login.elixir-czech.org/oidc/jwk
       - LOG_FORMAT=json
     volumes:
-      - ./certs/ca.crt:/certs/ca.crt
-      - ./certs/client.crt:/certs/client.crt
-      - ./certs/client.key:/certs/client.key
-      - ./certs/proxy.crt:/certs/proxy.crt
-      - ./certs/proxy.key:/certs/proxy.key
+      - proxy_certs:/certs
       - ./users.csv:/users.csv
       - ./keys:/keys
     ports:
       - "8000:8000"
       - "8001:8001"
 
+  tests:
+    image: golang:${GOLANG_VERSION:-1.18}
+    container_name: s3proxy-tests
+    profiles:
+      - test
+    command:
+      - "/bin/sh"
+      - "-c"
+      - "cd /app; echo 'Running go ${GOLANG_VERSION:-1.18} tests';
```

Comment:
```suggestion
      - "cd /app; echo 'Running go ${GOLANG_VERSION:-1.19} tests';
```

Reply:
1.19 broke something in the linting so I'll update golang versions in a separate PR :) don't want this one to grow out of hand...